### DISCUSSION TITLE
GitHub Account Hacked and Access Removed, Profile Invisible

### DISCUSSION THREAD

**Discussion by Rudra-241** [community/community#127386] [Other Feature Feedback, Questions, & Ideas]
My friend @AishikTokdar's GitHub Account got hacked recently. One morning he suddenly saw a series emails stating that the password was changed first, then both his emails were removed from the account.
He created a Support Ticket the same day : 2819995. He is yet to receive a response from your side.
He has no other way to contact you and I am writing this here on his behalf, so that you guys look into his case 

Some Account Details - 
Username: AishikTokdar
Primary Email : aishiktokdar@gmail.com

**Comment by ata-hastam** [comment]
Visit https://github.com/orgs/community/discussions/127293

**Comment by ebndev** [comment]
Hi @Rudra-241,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
